### PR TITLE
AB#109483 filter project - add new endpoint for getting all available statuses

### PR DESCRIPTION
**Pull request by jordan-beard**
image

### CODE REVIEW

File: Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectGetStatusesTests.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+		IProjectStatusesDataQuery dataQuery = new ProjectStatusesDataQuery(_context);
+		IProjectGetStatusesQuery query = new ProjectGetStatusesQuery(dataQuery);
+
+		_legacyProjectController = new LegacyProjectController(Mock.Of<ILegacyProjectGetQuery>(), Mock.Of<ILegacyProjectListGetQuery>(),
+			query, Mock.Of<ILegacyProjectUpdateCommand>());
+	}
+
+	[Fact]
+	public async Task ProjectExists___ProjectReturned()
```
[the remark this answers is not in the document]
Comment (dneed-nimble) [reply Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectGetStatusesTests.cs:34]: Understanding we'd probably have wider issues if this was the case so could see an argument against writing these.

**Review by dneed-nimble — APPROVED** [review]
Approved with test suggestion

File: Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/ProjectStatusesDataQueryTests.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+			var results = await _subject.Execute();
+
+			Assert.Multiple(
+				() => Assert.Equal(status, results.First()),
+				() => Assert.Single(results)
+			);
+		}
+
+		[Fact]
```
Comment (jordan-beard) [inline Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/ProjectStatusesDataQueryTests.cs:44]: @dneed-nimble I did one of those here. 
Though, happy to put another one in.

Comment (dneed-nimble) [reply Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/ProjectStatusesDataQueryTests.cs:44]: Happy to resolve

**Pull request merged**